### ISSUE TITLE
Travis CI in stable branch is broken

### ISSUE THREAD

**Issue by cutwater** [ansible/galaxy_ng#396]
```
    Traceback (most recent call last):
      File "/usr/local/bin/rq", line 8, in <module>
        sys.exit(main())
      File "/usr/local/lib/python3.7/site-packages/click/core.py", line 829, in __call__
        return self.main(*args, **kwargs)
      File "/usr/local/lib/python3.7/site-packages/click/core.py", line 782, in main
        rv = self.invoke(ctx)
      File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1259, in invoke
        return _process_result(sub_ctx.command.invoke(sub_ctx))
      File "/usr/local/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
        return ctx.invoke(self.callback, **ctx.params)
      File "/usr/local/lib/python3.7/site-packages/click/core.py", line 610, in invoke
        return callback(*args, **kwargs)
      File "/usr/local/lib/python3.7/site-packages/rq/cli/cli.py", line 77, in wrapper
        cli_config = CliConfig(**kwargs)
      File "/usr/local/lib/python3.7/site-packages/rq/cli/helpers.py", line 214, in __init__
        self.worker_class = import_attribute(worker_class)
      File "/usr/local/lib/python3.7/site-packages/rq/utils.py", line 128, in import_attribute
        module = importlib.import_module(module_name)
      File "/usr/lib64/python3.7/importlib/__init__.py", line 127, in import_module
        return _bootstrap._gcd_import(name[level:], package, level)
      File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
      File "<frozen importlib._bootstrap>", line 983, in _find_and_load
      File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
      File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
      File "<frozen importlib._bootstrap_external>", line 728, in exec_module
      File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
      File "/usr/local/lib/python3.7/site-packages/pulpcore/tasking/worker.py", line 15, in <module>
        django.setup()
      File "/usr/local/lib/python3.7/site-packages/django/__init__.py", line 19, in setup
        configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
      File "/usr/local/lib/python3.7/site-packages/django/conf/__init__.py", line 79, in __getattr__
        self._setup(name)
      File "/usr/local/lib/python3.7/site-packages/django/conf/__init__.py", line 66, in _setup
        self._wrapped = Settings(settings_module)
      File "/usr/local/lib/python3.7/site-packages/django/conf/__init__.py", line 157, in __init__
        mod = importlib.import_module(self.SETTINGS_MODULE)
      File "/usr/lib64/python3.7/importlib/__init__.py", line 127, in import_module
        return _bootstrap._gcd_import(name[level:], package, level)
      File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
      File "<frozen importlib._bootstrap>", line 983, in _find_and_load
      File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
      File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
      File "<frozen importlib._bootstrap_external>", line 728, in exec_module
      File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
      File "/usr/local/lib/python3.7/site-packages/pulpcore/app/settings.py", line 73, in <module>
        plugin_app_config = entry_point.load()
      File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2442, in load
        self.require(*args, **kwargs)
      File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2465, in require
        items = working_set.resolve(reqs, env, installer, extras=self.extras)
      File "/usr/lib/python3.7/site-packages/pkg_resources/__init__.py", line 791, in resolve
        raise VersionConflict(dist, req).with_context(dependent_req)
    pkg_resources.ContextualVersionConflict: (attrs 20.1.0 (/usr/local/lib/python3.7/site-packages), Requirement.parse('attrs<20,>=19.3.0'), {'galaxy-importer'})
```

**Comment by awcrosby** [comment]
I released `galaxy-importer==0.2.8rc12` which includes commit https://github.com/ansible/galaxy-importer/commit/a3cd9263440aa15ec8ba3462af7aa3aa65ebcef8 that allows for `attrs==2.1.0`, but that is not getting picked up in this part of the CI

**Comment by cutwater** [comment]
Resolved by #399
